plugin/build.gradle

```diff
@@ -58,7 +58,6 @@ dependencies {
     implementation project(':opensearch-ml-algorithms')
     implementation project(':opensearch-ml-search-processors')
     implementation project(':opensearch-ml-memory')
-    compileOnly "com.google.guava:guava:32.1.3-jre"
```

Comment:
Don't we need to do the same like this: https://github.com/opensearch-project/anomaly-detection/pull/1473/files ?

Reply:
We need to import the dependency during runtime using `implementation` here

Reply:
Yeah, we have it in the following.